src/components/binding/binding.js

```diff
@@ -150,13 +162,16 @@ genotet.BindingView = function(viewName, params) {
     .on('genotet.gene', function(event, data) {
       this.data.tracks[data.trackIndex].fileName = data.fileName;
       this.loader.loadFullTrack(data.trackIndex, data.fileName,
-        this.data.chr);
+        this.data.chr, false);
     }.bind(this));
 
   $(this.loader)
     .on('genotet.chr', function(event, chr) {
       this.panel.updateChr(chr);
     }.bind(this))
+    .on('genotet.addTrack', function() {
+      this.loader.loadBindingList();
```

Comment:
Will binding list ever be changed after a track is added? I don't see how binding list update is related to addTrack.

If you just want to refresh the list every time the panel is shown, then this could be requested from panel.